### PR TITLE
[wallet] Dispatch event on tx history fetching failure

### PR DESCRIPTION
**Pull request by rasom**
needed for https://github.com/status-im/status-react/issues/11502 

Currently if `controlCommand` fails it will be restarted infinitely.
If `controlCommand` failed three times in a row for any reason we dispatch an event about failure. There might be different reasons but usually it is either bad connection or RPC endpoint which doesn't allow history requests.

status: ready

### CODE REVIEW

File: services/wallet/commands.go [hunk 1]
```diff
@@ -830,6 +852,19 @@ func (c *controlCommand) Run(parent context.Context) error {
 	return err
 }
 
+func (c *controlCommand) NewError(err error) bool {
```
Comment (cammellos) [inline services/wallet/commands.go:855]: Why 3 exactly? could you add a description of the behavior that we want to achieve please?

Comment (rasom) [reply services/wallet/commands.go:855]: sure

Comment (rasom) [reply services/wallet/commands.go:855]: 3 is arbitrary here

Comment (rasom) [reply services/wallet/commands.go:855]: @cammellos added two sentences to PRs description, let me know if that's enough

**Review by cammellos — APPROVED** [review]

**Pull request merged**